decidim-conferences/app/cells/decidim/conferences/photo/show.erb

```diff
@@ -1,26 +1,19 @@
-<div class="column">
-  <div class="card" data-open="media-picture-<%= index %>">
-    <div class="picture__content">
-      <%= image_thumb %>
-        <strong class="card__title"><%= title %></strong>
-      <div class="text-small">
-        <%= short_description %>
-      </div>
-    </div>
-  </div>
-  <div id="media-picture-<%= index %>" class="reveal picture__modal" data-reveal>
-    <div class="reveal__header">
-      <h3 class="reveal__title"><%= t("conferences.photo.show.photo", scope: "decidim") %></h3>
-    </div>
-    <%= image_big %>
-    <h5>
-      <strong><%= title %></strong>
-    </h5>
-    <div class="text-small">
-      <%= decidim_sanitize_editor description %>
+<div class="conference__media-photo" data-conference-media-photo data-dialog-open="media-picture-<%= index %>">
```

Comment:
At staging I cannot open this modal.

This element is also not focusable using keyboard/tab navigation.